lisa/sut_orchestrator/azure/features.py

```diff
@@ -182,6 +185,191 @@ def is_over_nd(self) -> bool:
         dmesg = self._node.tools[Dmesg]
         return "hvnd_try_bind_nic" in dmesg.get_output()
 
+    def setup_rdma(self) -> None:
+        node = self._node
+        # Dependencies
+        kernel = node.tools[Uname].get_linux_information().kernel_version_raw
+        ubuntu_required_packages: List[str] = [
+            "build-essential",
+            "numactl",
+            "rpm",
+            "libnuma-dev",
+            "libmpc-dev",
+            "libmpfr-dev",
+            "libxml2-dev",
+            "m4",
+            "byacc",
+            "python-dev",
+            "python-setuptools",
+            "tcl",
+            "environment-modules",
+            "tk",
+            "texinfo",
+            "libudev-dev",
+            "binutils",
+            "binutils-dev",
+            "selinux-policy-dev",
+            "flex",
+            "libnl-3-dev",
+            "libnl-route-3-dev",
+            "libnl-3-200",
+            "bison",
+            "libnl-route-3-200",
+            "gfortran",
+            "cmake",
+            "libnl-3-dev",
+            "libnl-route-3-dev",
+            "libsecret-1-0",
+            "dkms",
+            "python-setuptools",
+        ]
+        redhat_required_packages: List[str] = [
+            "git",
+            "zip",
+            "python3",
+            "kernel-rpm-macros",
+            "gdb-headless",
+            "python36-devel",
+            "elfutils-libelf-devel",
+            "rpm-build",
+            "make",
+            "gcc",
+            "tcl",
+            "tk",
+            "gcc-gfortran",
+            "tcsh",
+            "kernel-devel",
+            "kernel-modules-extra",
+            "createrepo",
+            # "python-devel",
+            "libtool",
+            "fuse-libs",
+        ]
+        if isinstance(node.os, CentOs):
+            node.execute(
+                "yum install -y https://partnerpipelineshare.blob.core.windows.net"
+                f"/kernel-devel-rpms/kernel-devel-{kernel}.rpm",
+                sudo=True,
+            )
+
+        if isinstance(node.os, Redhat):
+            if node.os.information.version.major == 7:
+                redhat_required_packages.append("python-devel")
+            else:
+                redhat_required_packages.append("python3-devel")
+                redhat_required_packages.append("python2-devel")
+            node.os.install_packages(redhat_required_packages)
+        elif isinstance(node.os, Ubuntu):
+            node.os.install_packages(ubuntu_required_packages)
```

Comment:
What about else? Raise an exception for other unsupported OS. So we can see it in log clearly, and may support them later.